modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/styles/components/_radio.scss

```diff
@@ -0,0 +1,27 @@
+.radio-card {
+	border: 1px solid rgba($color: #000, $alpha: 0.1);
+	cursor: pointer !important;
+	gap: 0.75rem;
+
+	& > div {
+		> div {
+			label {
+				cursor: pointer;
+			}
+		}
+	}
+
+	.radio-card-label {
+		gap: 0 !important;
+	}
+
+	.custom-control-label {
+		&::before {
+			top: 0.3rem;
+		}
+
+		&::after {
+			top: 0.3rem;
+		}
+	}
+}
```

Comment:
We need to use our pattern, to start all custom styles with cp- preffix

Reply:
Ok, done!